### DISCUSSION TITLE
Support for motioneye dev (latest)

### DISCUSSION THREAD

**Discussion by e1ace** [tteck/Proxmox#1348] [Ideas]
The standard motioneye has not been updated in almost 2 years. The dev branch has a continued development.

https://github.com/motioneye-project/motioneye/tree/dev has details and installation instructions.

It would be great if there was an update to the scripts to support (or possibly choose during installation) the latest/current version.

**Comment by tteck** [comment]
Done!

**Reply by ChrisGoudt** [reply]
Seems to be an issue with the new version, when doing a new install I get an error:

[ERROR] in line 22: exit code 1: while executing command "$@" > /dev/null 2>&1

If you need more information or logs, feel free to let me know.

Also, when using Ubuntu 22 (04 or 10), there is an issue with flickering of the camera screen, this is a known issue in the stable motioneye version, which can be resolved by wget http://ftp.de.debian.org/debian/pool/main/libm/libmicrohttpd/libmicrohttpd12_0.9.72-2_amd64.deb and sudo dpkg -i libmicrohttpd12_0.9.72-2_amd64.deb. Might be something you want to add to the script.

Haven't tested if this problem also occurs with Debian versions or the dev branch version of motioneye.

**Comment by tteck** [comment]
> Seems to be an issue with the new version, when doing a new install I get an error:
> 
> [ERROR] in line 22: exit code 1: while executing command "$@" > /dev/null 2>&1
> 
> If you need more information or logs, feel free to let me know.

`silent()` function is defined (line 22), which takes a command as an argument and runs it in the background, directing standard output and standard error to `/dev/null` (the null device).
To troubleshoot the error message "`[ERROR] in line 22: exit code *: while executing command "$@" > /dev/null 2>&1`" run the script in verbose mode to identify the actual problem.

No issues here

```
    __  ___      __  _                           
   /  |/  /___  / /_(_)___  ____  ___  __  _____ 
  / /|_/ / __ \/ __/ / __ \/ __ \/ _ \/ / / / _ \
 / /  / / /_/ / /_/ / /_/ / / / /  __/ /_/ /  __/
/_/  /_/\____/\__/_/\____/_/ /_/\___/\__, /\___/ 
                                    /____/       
Using Advanced Settings
Using Distribution: debian
Using debian Version: 11
Using Container Type: 1
Using Root Password: Automatic Login
Container ID: 113
Using Hostname: motioneye
Using Disk Size: 8
Allocated Cores: 2
Allocated RAM: 2048
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a Motioneye LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 113 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.253 
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.113.3
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Installed Motion
 ✓ Installed FFmpeg
 ✓ Installed Python3-pip
 ✓ Installed MotionEye
 ✓ Created Service
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!

Motioneye should be reachable by going to the following URL.
         http://192.168.86.253:8765 
```
![Screenshot 2023-04-29 7 22 43 AM](https://user-images.githubusercontent.com/21342670/235300050-5fe60d6b-1668-4e41-a755-428dd15780c8.png)

**Reply by ChrisGoudt** [reply]
Yep, already was looking into what was causing it, seems to be an issue when installing it on Ubuntu.

<img width="548" alt="motioneye" src="https://user-images.githubusercontent.com/50422819/235300874-291ae54d-3326-4765-8ba1-86cdc4e6c524.png">
